### PR TITLE
Fixed taxes not applied to Shipping Cost #4350

### PR DESCRIPTION
**Pull request by devansh-webkul**
**BUGS:**
Link: https://github.com/bagisto/bagisto/issues/4350

### CODE REVIEW

**Comment by devansh-webkul** [comment]
Hi @rahulshukla-webkul,

This is a tax-related PR, please let me know. If any improvement is needed.

**Review by ghermans — CHANGES_REQUESTED** [review]
Since not all shipping carriers provide prices without tax included, i would suggest to add a checkbox at the shipping method configuration in admin to enable / disable tax calculation.

**Comment by devansh-webkul** [comment]
Hi @ghermans,

The mentioned changes have been done. Please let me know if any improvement is required.

File: packages/Webkul/Shipping/src/Config/system.php [hunk 1]
```diff
@@ -86,6 +93,13 @@
                 'validation'    => 'required',
                 'channel_based' => false,
                 'locale_based'  => true,
+            ], [
+                'name'          => 'is_calculate_tax',
+                'title'         => 'Calculate Tax',
```
Comment (ghermans) [inline packages/Webkul/Shipping/src/Config/system.php:98]: Please change this to: Calculate tax on checkout

**Review by ghermans — CHANGES_REQUESTED** [review]

File: packages/Webkul/Shipping/src/Config/system.php (hunk 1 above)
Comment (devansh-webkul) [reply packages/Webkul/Shipping/src/Config/system.php]: Added the missing translation.

**Comment by devansh-webkul** [comment]
Hi @ghermans,

The mentioned changes have been done.

**Pull request merged**